tools/homework3.sh

```diff
@@ -0,0 +1 @@
+find . -name "test.txt" | xargs grep 'hello'ls ./test.txt | grep "hello"
```

Comment:
@al14085 Did it match the homework content? btw, Is there any better way?

Reply:
I dont match this homework. I misunderstood. I will modify.